content/da/faq.md

```diff
@@ -1,34 +1,32 @@
 
-## How can I contribute?
+## Hvordan kan jeg bidrage?
 
-Everyone can help. io.js adheres to a [code of conduct](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-of-conduct), and contributions, releases, and contributorship are under an [open governance](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme) model.
+Alle kan hjælpe. io.js holder sig til et [adfærdskodeks](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-of-conduct), og bidrag og udgivelser er underlagt en [åben forvaltningsmodel](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme).
 
-To get started, there are open [ discussions on GitHub](https://github.com/iojs/io.js/issues), and we'd love to hear your feedback.
-Becoming involved in discussions is a good way to get a feel of where you can help out further. If there is
-something there you feel you can tackle, please [make a pull request](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-contributions).
+For at komme igang, er der åbne [ diskussioner på GitHub](https://github.com/iojs/io.js/issues) og vi vil meget gerne høre dine tilbagemeldinger.
+At blive en del af diskussioner er en god måde at få føling for, hvordan du kan hjælpe yderligere til. Hvis der er noget der, du føler, du kan håndtere, så [lav venligst en pull request](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-contributions).
 
-In addition, using [Nodebug.me](http://nodebug.me/) is a good way to help Triage the issues in the backlog.
+Udover det, er det at bruge [Nodebug.me](http://nodebug.me/) en god måde at hjælpe med at vurdere vigtigheden af eksisterende issues.
 
-## Where do discussions take place?
+## Hvor finder diskussioner sted?
 
-There is an #io.js channel on Freenode IRC. We keep logs of the channel [here](http://logs.libuv.org/io.js/latest).
+Der er en #io.js-kanal på Freenode IRC. Vi holder logs fra kanalen [her](http://logs.libuv.org/io.js/latest).
 
-## What is open source governance?
+## Hvad er open source-forvaltning?
 
-Open source governance advocates the application of the philosophies of the open source and open content movements in order to enable any interested party to add to the creation of the end product, as with a wiki document. Legislation is democratically opened to the general citizenry, employing their collective wisdom to benefit the decision-making process and improve democracy. [[source]](https://en.wikipedia.org/wiki/Open-source_governance)
\ No newline at end of file
+Open source-forvaltning taler for at bruge filosifierne fra open source- og open content-bevægelser for at give enhver interesseret part mulighed for at tilføje til skabelsen af slutproduktet, så vel som et wiki-dokument. Lovgivning er demokratisk åbnet op for generelle borgere, der bruger deres kollektive viden til at fordel for beslutningsprocessen og for at forbedre demokratiet. [[kilde]](https://en.wikipedia.org/wiki/Open-source_governance)
```

Comment:
> kollektive viden til at fordel for beslutningsprocessen og for at forbedre demokratiet

kollektive viden til ~~at~~ fordel for beslutningsprocessen og ~~for~~ **til** at forbedre demokratiet